AndelaOSP/art-dashboard
# chore(sidemenu): wrap components in RoutesComponent in SideMenuComponent

## What does this PR do?
Display the side menu in the app

## Description of Task to be completed?
Wrap components in the `SideMenuComponent`. This makes these components display the menu sidebar. The wrapped components are:

- Dashboard component
- AssetTypes component
- AddAsset component 

## How should this be manually tested?

Visit the following routes:
- `/`
- `/asset_types`
- `/asset/add`

These routes should display a sidebar

## What are the relevant pivotal tracker stories?
[#157997062](https://www.pivotaltracker.com/story/show/157997062)

## Any background context you want to add?
N/A

## Important notes
N/A

## Packages installed
N/A

## Deployment note
N/A

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
![screen shot 2018-05-31 at 18 24 07](https://user-images.githubusercontent.com/24937453/40797363-d99e0b60-64ff-11e8-9690-6101fe043f47.png)
